lib/archethic/contracts/worker.ex

```diff
@@ -482,4 +294,48 @@ defmodule Archethic.Contracts.Worker do
         {:error, :not_enough_funds}
     end
   end
+
+  # no one use the return value
+  defp do_execute_trigger_and_handle_result(
```

Comment:
Maybe this one also can be split into two: trigger execution and validation of the flow like this one.